### ISSUE TITLE
Simulation results depend on connection order to contact block

### ISSUE THREAD

**Issue by AlphaScorpii-dev** [oestersoetebier/IdealizedContact#27]
I noticed that the simulation results were different according to how I connect objects to a contact block.
This actually even appears in the examples (for instance, example15), when inverting the 2 connections to the convex bodies gives actually different results.
In the case of example16, the simulation actually fails after inversion.

Tested on Dymola 2015FD01 and 2016FD01

**Comment by oestersoetebier** [comment]
This is correct, but actually not a bug. At least in those case where contact surfaces are different from each other. Contact block gets geometry information from the contact interface. If you invert the connections it appears like inverting the geometry information but not the surface type. This is why it says: "first surface mentioned must be connected to port 1" in the parameter dialog.
Didn't you get an assert message?
As far as Example 15 is concerned, both surface types are identical. In those cases inverting the connection should work. What happens is that the reference (master) for calculation is changed. This particular convexToConvex contact combination relies on numeric contact point calculations. Therfore, changing the intial conditions by changing the reference may lead to differences. I will use Example 15 for further investigations.

**Comment by AlphaScorpii-dev** [comment]
Thanks for your reply. I was aware that the contact model was relying on the object type connected to the port. That's why I gave 2 examples with the same kind of objects on both sides. Then, the contact model would be Ellipse/Ellipse, or Convex/Convex, and that wouldn't matter anyway.
